token.go

```diff
@@ -5,218 +5,262 @@
 package gocql
 
 import (
-	"bytes"
-	"crypto/md5"
-	"fmt"
-	"math/big"
-	"sort"
-	"strconv"
-	"strings"
-
-	"github.com/gocql/gocql/internal/murmur"
+    "bytes"
```

Comment:
**nitpick**: It seems that in some files, you have changed tabs to spaces. Could you please keep tabs?